src/main/java/core/basesyntax/db/BalanceStorage.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.db;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import javax.management.openmbean.KeyAlreadyExistsException;
+
+public class BalanceStorage<K, V> implements Dao<K,V> {
```

Comment:
for implementations of interfaces I prefer adding distinguishing feature to the name of implementing class instead of ...Impl. It makes code more readable and helps to distinguish different implementations by its key feature. 
![image](https://user-images.githubusercontent.com/37250267/225345524-75a406af-fac8-473f-bf37-474b5d64d0fb.png)